### PR TITLE
Add dutch/nederlands (nl) translation of the airrohr instructions.

### LINKED ISSUE #128: Dutch translation of airrohr and dnms building instructions?
The dutch language page of sensor.community/nl, 'build your own sensor' points to english airrohr and dnms instructions.
Would it be possible to add a dutch version of both instructions?
I can fork the english version and translate them but I noticed there is no NL folder on github to store these translations

### PR DESCRIPTION
**Pull request by bertrik**
This adds the dutch/nederlands (nl) translation for the airrohr instructions.
I've used the same file names as the english translation.

This adds files in content/airrohr/nl, not sure if other references need to be updated too.

### CODE REVIEW

File: content/airrohr/nl/02-assemble.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+See __firmware flasher__ section.
+
+### NodeMCU v3
+Let op: onze instructies gaan over versie 3 van de NodeMCU. Deze kan je herkennen aan de pinnen VU en G (zie tekening).
+
+<img src="../docs/airrohr-wiring-sds011-bme280.jpg" style="width:40%; margin-top: 3em"/>
+<small>Copyright: roman-minyaylov, MIT License</small>
+
+
+<img src="../docs/nodemcu-v3-bme280.jpeg" style="margin-top: 1em"/>
```
Comment (ohheyitsdave) [inline content/airrohr/nl/02-assemble.md]: The path is not correct, here is the correct one:
``

File: content/airrohr/nl/02-assemble.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+SDS011 Pin 2 -> Pin D2 / GPIO4
+SDS011 Pin 3 -> GND
+SDS011 Pin 4 -> niet aangesloten
+SDS011 Pin 5 -> VU (NodeMCU v3) / VIN (NodeMCU v1,v2)
+SDS011 Pin 6 -> niet aangesloten
+SDS011 Pin 7 -> niet aangesloten
+```
+
+### Solderen van de BME280
+<img src="../docs/solder-a-bme-280.jpeg" style="width:49%; padding-right: 0.5em" class="items-center"/>
```
Comment (ohheyitsdave) [inline content/airrohr/nl/02-assemble.md]: ``

File: content/airrohr/nl/02-assemble.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+<img src="../docs/solder-bme-280.jpeg" style="width:49%;">
```
Comment (ohheyitsdave) [inline content/airrohr/nl/02-assemble.md]: ``

File: content/airrohr/nl/02-assemble.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+VIN -> Pin 3V3 (3.3V)
+GND->  GND/G
+SDA -> PIN D3
+SCL -> Pin D4
+```
+
+### Alles bij elkaar binden
+
+ ##### Bind de NodeMCU en de SDS011 samen
+<img src="../docs/tie-air-quality-sensor-together.jpeg"/>
```
Comment (ohheyitsdave) [inline content/airrohr/nl/02-assemble.md]: ``

File: content/airrohr/nl/02-assemble.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+Gebruik een kabelbinder om de NodeMCU (ESP8266) en de SDS011-sensor aan elkaar te verbinden zodat de WiFi-antenne van de sensor afwijst
+
+ ##### Verbind de flexibele slang
+ <img src="../docs/sds011-with-tube.jpeg" style="width:49%; padding-right: 0.5em"/>
+ <img src="../docs/bme280-tied-to-tube.jpeg" style="width:49%;">
```
Comment (ohheyitsdave) [inline content/airrohr/nl/02-assemble.md]: ``
``

File: content/airrohr/nl/02-assemble.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+* Gebruik nog een kabelbinder om de BME280-temperatuursensor aan de slang te binden
+* Haal de USB-kabel door de pijp. Monteer de SDS011 met de NodeMCU naar boven wijzend en de ventilator naar beneden
+
+ 
+ ##### Stop de sensor in de pijp
+* Duw de onderdelen in de pijp, zodat ze stevig vast zitten
+* De USB-kabel, flexibele slang en de BME280 zouden uit de pijp moeten steken
+* Duw het andere pijpdeel op de eerste
+
+<img src="../docs/sds011-jammed-into-tube.jpeg"/>
```
Comment (ohheyitsdave) [inline content/airrohr/nl/02-assemble.md]: ``

File: content/airrohr/nl/02-assemble.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+ ##### Afwerking
+* Positioneer de temperatuursensor op de flexibele slang, zodat deze bij het uiteinde van de pijp zit.
+* Knip de flexibele slang af aan het eind van de pijp
+* Optioneel: je kan de open einden van de pijp bedekken met een fijn gaas. Zo kan de lucht circuleren maar hou je insecten buiten.
+ 
+<img src="../docs/position-bme280.jpeg"/>
```
Comment (ohheyitsdave) [inline content/airrohr/nl/02-assemble.md]: ``

**Review by ohheyitsdave — CHANGES_REQUESTED** [review]
Hi @bertrik 
thanks for you contribution, looks good, only some paths to the pictures are not correct, the path `/airrohr/`was missing.
Best regards
David

**Comment by bertrik** [comment]
Hi, I've applied the suggested updates.

**Review by ohheyitsdave — APPROVED** [review]
Looks good @bertrik. Gonna merge it now.
Thanks again for your contribution 😊

**Pull request merged**